server/src/main/scala/io/delta/standalone/internal/DeltaSharingHistoryManager.scala

```diff
@@ -33,7 +33,7 @@ object DeltaSharingHistoryManager {
    */
   private val POTENTIALLY_UNMONOTONIZED_TIMESTAMPS = 100
 
-  private[internal] def getCommitsSafe(
+  def getCommitsSafe(
       logStore: LogStore,
```

Comment:
I meant make it public in DeltaHistoryManager.
We may not need DeltaSharingHistoryManager then?

Reply:
Do you mean DeltaHistoryManager? https://sourcegraph.com/github.com/delta-io/connectors/-/blob/standalone/src/main/scala/io/delta/standalone/internal/DeltaHistoryManager.scala?L166

We need to make it public, and make another release, then we don't need DeltaSharingHistoryManager. Which, takes time.

Reply:
Ok, so maybe add a todo for now to make remove DeltaSharingHistoryManager once the DeltaSharingManager changes are in and a new release it cut?

Reply:
sg, added an issue as well https://github.com/delta-io/delta-sharing/issues/144